Agrume/Agrume.swift

```diff
@@ -14,20 +14,20 @@ public protocol AgrumeDataSource {
   ///
   /// - Parameter index: The index (collection view item) being displayed
   /// - Parameter completion: The completion that returns the image to be shown at the index
-	func imageForIndex(index: Int, completion: (UIImage?) -> Void)
+	func imageForIndex(_ index: Int, completion: (UIImage?) -> Void)
```

Comment:
I guess the Swift 3 way would be to rename the method to `fun image(forIndex index: Int, …)`